_pages/en_US/sd-card-setup.md

```diff
@@ -63,7 +63,7 @@ If the test shows any other results, your SD card may be corrupted or damaged an
    <h2>Linux</h2>
 </noscript>
 
-## Section I - Determining which slot your SD card is in
+## Section I - Determining which slot your SD card is in and Formatting the card
```

Comment:
```suggestion
## Section I - Formatting your SD card
```

There, made a proper suggestion on it ;P

Is there really anything holding this back besides this heading? If not then we should probably adjust it if needed and merge it if these instructions are better